content/hi/docs/tutorials/kubernetes-basics/deploy-app/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: ऐप डिप्लॉय करें
```

Comment:
Can you use "Application deployment" or "Deploy application" to localise in hindi?